circuit_tutorials/plonky2/merkle_tree/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "sindri-script"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
```

Comment:
This Cargo.toml needs updated.  A lot of plonky2 dependencies are used in `src/main.rs`